packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -1059,6 +1060,124 @@ curl -v -X POST \
 }
 ```
 
+#### Add Smart Card Identity Provider
+
+
+Adds a new `Smart Card` type IdP to your organization
+
+##### Request Example
+
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "type": "X509",
+  "status": "ACTIVE",
+  "features": [],
+  "name": "Smart Card IDP Name",
+  "protocol": {
+    "type": "MTLS",
+    "credentials": {
+      "trust": {
+        "revocation": "CRL",
+        "revocationCacheLifetime": 2880,
+        "issuer": "CN=Test Smart Card, OU=Test OU, O=Test O, C=US",
+        "kid": "45dec5ff-8cdc-48c0-85fe-a4869f1753dc"
```

Comment:
1. Where does the developer get the "kid" from? 
2. We require the admins to upload the entire CA chain (Root, Intermediates, Issuing CA). Does this "kid" correspond to the CA chain? the "issuer" field only shows the Root CA information 
3. How do developers upload the CA chain? We will need to document that as well.